server/util/tracer.js

```diff
@@ -0,0 +1,83 @@
+const { registerInstrumentations } = require('@opentelemetry/instrumentation');
+const opentelemetry = require('@opentelemetry/api');
+
+// Not functionally required but gives some insight what happens behind the scenes
+// const { diag, DiagConsoleLogger, DiagLogLevel } = opentelemetry;
+// diag.setLogger(new DiagConsoleLogger(), DiagLogLevel.INFO);
+
+const { AlwaysOnSampler, SamplingDecision, SimpleSpanProcessor } = require('@opentelemetry/sdk-trace-base');
+const { GraphQLInstrumentation } = require('@opentelemetry/instrumentation-graphql');
+const { OTLPTraceExporter } = require('@opentelemetry/exporter-trace-otlp-proto');
+const { ConsoleSpanExporter, NodeTracerProvider } = require('@opentelemetry/sdk-trace-node');
+const { Resource } = require('@opentelemetry/resources');
+const { WinstonInstrumentation } = require('@opentelemetry/instrumentation-winston');
+const { SemanticResourceAttributes } = require('@opentelemetry/semantic-conventions');
+
+const Exporter = (process.env.EXPORTER || '').toLowerCase().startsWith('c') ? ConsoleSpanExporter : OTLPTraceExporter;
+const { ExpressInstrumentation } = require('@opentelemetry/instrumentation-express');
+const { HttpInstrumentation } = require('@opentelemetry/instrumentation-http');
+
+const otlpDisabled = process.env?.OTEL_SDK_DISABLED || false;
```

Comment:
The SDK should actually see this env var and disable things, we technically shouldn't need to wrap setup in a separate condition.

https://github.com/open-telemetry/opentelemetry-java/blob/main/sdk-extensions/autoconfigure/README.md#disabling-opentelemetrysdk

Reply:
testing this out. I wrapped it to prevent initialization of those extra libs as I do have concerns about the performance implications of these libraries.

Reply:
Just using that ENV var does not work across all of the different collectors, it also doesn't disable the exporter... sticking with the wrapper for now.